src/main/java/io/pokerwars/bot/strategies/HandUtil.java

```diff
@@ -21,7 +21,7 @@
 
 public class HandUtil {
```

Comment:
I wounder if we could use an external library, this code is not the best

Reply:
I know but I don't think it make sense to spend too much time on integrating a library